app/javascript/modules/validate-forms.js

```diff
@@ -1,4 +1,4 @@
-// Example starter JavaScript for disabling form submissions if there are invalid fields
+// JavaScript to disable form submissions if there are invalid fields
```

Comment:
this comment change is in a separate commit;  I know this file started as a straight up cut and paste from Bootstrap documentation, but ... it's ours now.